packages/compass-components/src/hooks/use-confirmation.tsx

```diff
@@ -207,3 +209,32 @@ export const ConfirmationModalArea: React.FunctionComponent = ({
     </ConfirmationModalAreaMountedContext.Provider>
   );
 };
+
+export async function showPrompt(
```

Comment:
New thing: similar to showConfirmation allows to prompt the user in place without the need for all the extra state that's usually associated with this UI